framework/Element.js

```diff
@@ -43,41 +43,50 @@ export default class Element {
         }
     }
 
-    async getCoordinates(xOffset = null, yOffset = null) {
+    async getElementClickCoordinates(xOffset = null, yOffset = null) {
```

Comment:
I suggest not to rename this method as it is public, and it would be a breaking change.

Reply:
Reverted back.